rts/lra/lra-coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/Transaction.java

```diff
@@ -869,4 +873,14 @@ public boolean afterLRANotification() {
 
         return notifiedAll;
     }
+
+    static class Pair {
```

Comment:
Would not be good to name it in way to explain what is the purpose of the `Pair`? As it's a inline class it's expected to be used in the closed context of the class and the name can express what is that for. It could be like `LRANotificationPair` or `LRANotificationListenerData` or something similar.

Reply:
I agree, it's hard to read `Pair participant = ...`. I think this is `AfterLRAParticipant` which consists of required afterLRA URI and optional recoveryURI if it is also LRA participant in the normal sense.